src/Util/CsvImportUtil.php

```diff
@@ -241,20 +236,61 @@ public function setCallback(callable $callback)
      */
     public function getSeparators()
     {
+        if (count($this->separators) < 1) {
+            $this->setDefaultSeparators();
+        }
+
         return $this->separators;
     }
 
     /**
      * Set separators
      *
      * @param array $separators
+     *
+     * @throws \InvalidArgumentException
      */
     public function setSeparators(array $separators)
     {
+        foreach ($separators as $separator) {
+            if (!is_array($separator) || !$separator['value'] || !$separator['label']) {
+                throw new \InvalidArgumentException(
+                    'Each separator must be an array that contains "value" and "label" keys!'
+                );
+            }
+        }
+
         $this->separators = $separators;
     }
 
     /**
+     * Set default separators
+     */
+    private function setDefaultSeparators()
```

Comment:
this should be done in the constructor to always set a default value for the property